package.json

```
@@ -99,6 +99,7 @@
     "jest-mock-extended": "1.0.8",
     "nock": "12.0.3",
     "prettier": "1.19.1",
+    "prettier-default-config": "^0.2.3",
```

```suggestion
    "prettier-default-config": "0.2.3",
```
I think it's better to specify the exact version of the package :D you might get some lock file issues after changing it, so I suggest running `yarn install` post-change :)

---

It's definitely better because when the user installs the DX Scanner from NPM globally so it doesn't respect a lockfile.